server/controllers/finance/patientInvoice.js

```diff
@@ -225,6 +225,8 @@ function find(options) {
   filters.dateFrom('billingDateFrom', 'date');
   filters.dateTo('billingDateTo', 'date');
 
+  filters.subRequest('uuid', 'cash_item', 'invoice_uuid', 'cash_uuid');
```

Comment:
I don't think this is defined anymore.  It's breaking the tests.  Could you remove it?